zen/pkm/cli.py

```
@@ -0,0 +1,349 @@
+
+@pkm.command()
+@click.option("--config", "-c", type=click.Path(exists=True), help="Path to configuration file")
+def stats(config: Optional[str]):
+    """Show PKM statistics"""
+    config_path = Path(config) if config else None
+    pkm_config = PKMConfig.load(config_path)
+    storage = PKMStorage(pkm_config)
+    
+    stats = storage.get_statistics()
+    conversations = storage.list_conversations()
+    knowledge_entries = storage.list_knowledge_entries()
```

Ruff (0.13.2)

Local variable `knowledge_entries` is assigned to but never used

Remove assignment to unused variable `knowledge_entries`

(F841)